lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -307,8 +365,13 @@ export class NativeInteractionClient extends BaseInteractionClient {
             this.validateNativeResponse(response);
         } catch (e) {
             // Only throw fatal errors here to allow application to fallback to regular redirect. Otherwise proceed and the error will be thrown in handleRedirectPromise
```

Comment:
Do you know if we document that if an error here happens, the application needs to retry?  It feels like something we should handle for them.

Reply:
We fall back to the web flow if the native flow throws. cc @tnorling